release-engineering/role-handbooks/patch-release-team.md

```diff
@@ -256,6 +263,174 @@ For each cherry-pick request:
     PRs pending merge will be visible as rows distinct from the master
     branch in the [Tide Status](https://prow.k8s.io/tide).
 
+## Patch branch merge workflow and status tracking
+
+###  Rationale
+
+At any moment we typically have between a half dozen and a dozen
+cherry pick PRs pending on each branch.  Many of these are in a
+"cluster", picking the same change from master back to the various
+release branches.  These can be found per branch by a query such as:
+[is:open is:pr base:release-1.16
+label:do-not-merge/cherry-pick-not-approved](https://github.com/kubernetes/kubernetes/pulls?utf8=✓&q=is%3Aopen+is%3Apr+base%3Arelease-1.16+label%3Ado-not-merge%2Fcherry-pick-not-approved+).
+
+But these GitHub queries are point-in-time and do not show what's
+changed versus a prior run of the query.  Intermediate state must
+be externally recorded and updated for consistency.
+
+### Spreadsheets
+
+We maintain this evolving state in a set of spreadsheets, one
+spreadsheet per release branch.  These are privately shared amongst
+the patch release team because they contain information on inbound
+PRs that relate to security patches and target release date information
+that may be embargoed as per the Kubernetes security process (link).
+The use of the Google Docs platform means the team can simultaneously
+edit the state while also retaining a version history of changes
+for the team's future reference.
+
+![Image of a patch release team spreadsheet example](patch-release-team-spreadsheet-example.png)
+
+#### Time Ordered Rows
+
+The spreadsheets run in reverse chronological order with the newest
+patch release's activity at the top.
+
+Sections are made per patch release.
+
+A row is entered in a section per cherry pick pending ahead of that
+patch release.  Within each of these sections, cherry picks are
+listed in reverse PR number order, newest on top.  To add a notation
+for a newly inbound cherry pick, add a row above the current newest
+row.
```

Comment:
how often do pst members enter or modify new items?

Reply:
Depends. I guess less often, after we cut a patch, more often just before we cut a patch. I think I had times where I did check on cherry pick PR only once a week and I had times where I checked and updated cherry pick PRs more then once a day. Not sure how other's on the team handle that though.

Reply:
Over in https://github.com/kubernetes/community/pull/4081 I note we scrub things about weekly, but more often just before release.